rpcs3/Emu/CPU/CPUTranslator.h

```diff
@@ -3640,25 +3646,41 @@ class cpu_translator
 	template <typename T, typename = std::enable_if_t<std::is_same_v<llvm_common_t<T>, f32[4]>>>
 	static auto fre(T&& a)
 	{
+#ifdef ARCH_X64
 		return llvm_calli<f32[4], T>{"llvm.x86.sse.rcp.ps", {std::forward<T>(a)}};
+#else
```

Comment:
```suggestion
#elif defined(ARCH_ARM64)
```
It's better to keep it explicit what's specific to ARM arch.